integration/presto/src/main/prestosql/org/apache/carbondata/presto/PrestoFilterUtil.java

```diff
@@ -78,6 +78,8 @@ private static DataType spi2CarbondataTypeMapper(HiveColumnHandle columnHandle)
     HiveType colType = columnHandle.getHiveType();
     if (colType.equals(HiveType.HIVE_BOOLEAN)) {
       return DataTypes.BOOLEAN;
+    } else if (colType.equals(HiveType.HIVE_BINARY)) {
```

Comment:
I can see byte and float data type is also missing. can you add and test for it ?

Reply:
done